ietfdata/datatracker.py

```diff
@@ -1261,6 +1295,64 @@ def meeting_types(self) -> Iterator[MeetingType]:
         """
         return self._retrieve_multi("/api/v1/name/meetingtypename/", MeetingType)
 
+# =================================================================================================================================
+#   https://datatracker.ietf.org/api/v1/doc/relateddocument/?source=...      - documents that source draft relates to (references, replaces, etc)
+#   https://datatracker.ietf.org/api/v1/doc/relateddocument/?target=...      - documents that relate to target draft
+
+    def related_documents(self, 
+            source               : Optional[Document]         = None, 
+            target               : Optional[Document]         = None, 
+            relationship_type    : Optional[RelationshipType] = None) -> Iterator[RelatedDocument]:
+
+        url = "/api/v1/doc/relateddocument/"
+        if source is not None and target is not None and relationship_type is not None:
```

Comment:
This sequence of `if` statements is poor style. It's okay for now, but I trust you have a follow-on PR that will tidy up the code so it's no longer needed.

Reply:
Sequence of statements has now been shortened